docs/source/api/plugin/landing-pages.mdx

```diff
@@ -461,94 +461,7 @@ The default value is `dark`.
-
-The `ApolloServerPluginLandingPageGraphQLPlayground` plugin serves the [GraphQL Playground IDE](https://github.com/graphql/graphql-playground) as a landing page.
-
-The GraphQL Playground plugin lives in its own `@apollo/server-plugin-landing-page-graphql-playground` package. To use it, install this package and provide the plugin to the `ApolloServer` constructor:
-
-<MultiCodeBlock>
-
-```ts
-import { ApolloServer } from '@apollo/server';
-import {
-  ApolloServerPluginLandingPageGraphQLPlayground
-} from "@apollo/server-plugin-landing-page-graphql-playground";
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  plugins: [
-    ApolloServerPluginLandingPageGraphQLPlayground(),
-  ],
-});
-```
-
-</MultiCodeBlock>
-
-This plugin takes the same arguments as the [`renderPlaygroundPage` function](https://github.com/apollographql/graphql-playground/blob/apollo/packages/graphql-playground-html/src/render-playground-page.ts#L63) from the `graphql-playground-html` package (or specifically, the `@apollographql/graphql-playground-html` fork).
-
-[The table below](#options-2) mentions a few of the more common options and is not exhaustive.
-
-### Options
-
-<table class="field-table">
-  <thead>
-    <tr>
-      <th>Name /<br/>Type</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-
-<tbody>
-
-<tr>
-<td>
-
-###### `version`
-
-`string`
-</td>
-<td>
-
-By default, this plugin loads a specific npm version of `@apollographql/graphql-playground-react` from a CDN. The version is hard-coded in `apollo-server-core` and is typically incremented when new versions of the Playground fork are released (this is rare because the project is retired).
-
-You can specify a different version here.
-
-</td>
-</tr>
-
-<tr>
-<td>
-
-###### `endpoint`
-
-`string`
-</td>
-<td>
-
-By default, GraphQL Playground connects to a GraphQL server hosted at the same URL as Playground itself. To specify a different GraphQL endpoint, use this option.
-
-</td>
-</tr>
-
-<tr>
-<td>
-
-###### `settings`
-
-`Object`
-</td>
-<td>
-
-If specified, allows you to override the default values of GraphQL Playground's settings. Documentation of supported settings is available in [GraphQL Playground's README](https://github.com/graphql/graphql-playground#settings).
-
-</td>
-</tr>
-
-
-</tbody>
-</table>
+By default, Apollo Server 2 provided a GraphQL Playground landing page. For migration purposes, we've published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, a GraphQL Playground plugin compatible with Apollo Server 4. However, we aren't supporting this plugin with documentation or security updates. Instead, we recommend migrating to the actively maintained [Apollo Sandbox](/studio/explorer/sandbox) (the default landing page in Apollo Server 4) at your earliest convenience.
```

Comment:
Can we keep around (somehow) the mention that GraphQL Playground is officially retired?

Reply:
@glasser Is it possibly un-retired? This is a fairly recent comment.

https://github.com/graphql/graphql-playground/issues/1366#issuecomment-1062088978

Reply:
If I interpret this correctly, there's no longer a plan to *merge* the projects. But nobody is actually working on Playground: Rikki opened some issues earlier this year about the idea of continued work but is not planning to do that work personally. And the comment you linked is just saying Playground might be a thing in the future but just a wrapper for GraphiQL? In any case it doesn't seem to be super actively maintained now, and if it ever re-appears it will be quite different from the existing playground that our plugin wraps.  If it comes back to life they can make their own Apollo Server plugin :)

Reply:
Agreed! Done 👍